source/features/index.js

```diff
@@ -0,0 +1,181 @@
+import groupBy from 'lodash.groupby';
+import sortBy from 'lodash.sortby';
+
+export const features = {
+	/* GENERAL */
+	keyboardShortcuts: {
+		id: 'feature-keyboard-shortcuts',
+		category: 'general',
+		label: 'Enable keyboard shortcut to toggle Night Mode (Alt + M)',
```

Comment:
There's no good reason to turn this off.

Reply:
Ok